content/scripts/build.js

```diff
@@ -1062,7 +1071,7 @@ class Builder {
       doc,
       destinationDir,
       buildHtml: !this.options.buildJsonOnly,
-      titles: this.allTitles,
+      allTitles: this.allTitles,
```

Comment:
I renamed this because I find it mentally easier that it's called the famous "allTitles" as I hop over to the ssr code.